src/Entities/Entity.cpp

```diff
@@ -690,6 +690,33 @@ int cEntity::GetEnchantmentCoverAgainst(const cEntity * a_Attacker, eDamageType
 
 
 
+
+float cEntity::GetEnchantmentBlastKnockbackReduce()
+{
+	uint32_t MaxLevel = 0;
+
+	const cItem ArmorItems[] = { GetEquippedHelmet(), GetEquippedChestplate(), GetEquippedLeggings(), GetEquippedBoots() };
+
+	for (auto & Item : ArmorItems)
+	{
+		uint32_t Level = Item.m_Enchantments.GetLevel(cEnchantments::enchBlastProtection);
```

Comment:
Style: You can either use `unsigned` or `UInt32` but the `_t` versions shouldn't be used.